galsim/chromatic.py

```diff
@@ -804,50 +819,60 @@ def __init__(self, obj, waves, oversample_fac=1.0):
 
         # Make the objects between which we are going to interpolate.  Note that these do not have
         # to be saved for later, unlike the images.
-        objs = [ obj.evaluateAtWavelength(wave) for wave in self.waves ]
+        objs = [obj.evaluateAtWavelength(wave) for wave in self.waves]
 
         # Check the fluxes for the objects.  If they are unity (within some tolerance) then that
         # makes things simple.  If they are not, however, then we have to reset them to unity, and
         # modify the SED attribute, which now refers to the total flux at a given wavelength after
         # integrating over the whole light profile.
-        fluxes = np.array([ obj.getFlux() for obj in objs ])
+        fluxes = np.array([o.getFlux() for o in objs])
```

Comment:
Is this a rule?  That you can't use `obj` as a variable?  (a) Let's ignore that rule.  (b) `o` is a much worse variable name.